### ISSUE TITLE
2.2.1. binder not loading

### ISSUE THREAD

**Issue by demitch** [geoscixyz/Toolkit#123]
2.2.1. Search Case Study - Magnetic Data Visualization

![image](https://user-images.githubusercontent.com/39281824/51079768-8423a980-1682-11e9-8edf-0a050988c9b4.png)

**Comment by fourndo** [comment]
It worked on my side.
Can you try again?

**Comment by demitch** [comment]
weird, I just tried it again and it's still not working. Did you access through the Search Case study section ? (not 'The Apps' section)

aka, here:

![image](https://user-images.githubusercontent.com/39281824/51139303-552f4400-17f8-11e9-99a1-a5d1f9c9fa30.png)

**Comment by fourndo** [comment]
Yep from that page.
Doesn't make sense since they are pointing to the same url.
What happens when you click on the launch button?

**Comment by demitch** [comment]
@fourndo you're right, if I click 'launch binder' its all fine! Doesn't go if I click on the image link

**Comment by fourndo** [comment]
Fixed on #127
